src/components/common/AppBreadcrumbs.tsx

```diff
@@ -102,20 +123,64 @@ const AppBreadcrumbs = (props: Props) => {
     Boolean,
   );
   return (
-    <Breadcrumbs className={classes.root} separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
-      {crumbs.map((crumb, i) => (
-        <Crumb
-          key={crumb.name}
-          active={crumbs.length - 1 === i}
-          name={crumb.name}
-          href={crumb.href}
-          Icon={crumb.Icon}
-        />
-      ))}
-    </Breadcrumbs>
+    <Stack className={classes.root} direction="row" spacing={1}>
+      {viewer.relatedOwners && viewer.relatedOwners.length === 1 ? null : <AccountsButton viewer={viewer} />}
```

Comment:
If null `Breadcrumbs` is displayed at the beginning of the container. There is no spacing in Stack because of one element inside it in this case.  I use `Stack` as flex container to align `AccountsButton` and `Breadcrumbs` in row and vertically. 

<img width="737" alt="image" src="https://user-images.githubusercontent.com/74831191/211802610-e4e9da4e-255e-4b25-84a9-59323b1a4208.png">

Reply:
Seems if you'll remove the Stack and will put:

```typeescript
    <Breadcrumbs
      className={classes.breadcrumbs}
      separator={<NavigateNextIcon fontSize="small"/>}
      aria-label="breadcrumb"
    >
      {viewer.relatedOwners && viewer.relatedOwners.length === 1 ? null : <AccountSwitch viewer={viewer}/>}
      {crumbs.map((crumb, i) => (
      // the rest
```

It kind of alligns vertically. At least it seems so to me. 🤔 

<img width="964" alt="Screenshot 2023-01-11 at 9 45 42 AM" src="https://user-images.githubusercontent.com/989066/211836248-7c7b7ad9-1c34-4abd-a162-e877cab091ed.png">

Reply:
But do we need this button as part of `Breadcrumbs` with chevron-right after it? 🤔

Reply:
Good point! Didn't notice it. Let's than have the `Stack` since that makes the difference.